Feature/build active record

mcervantes71 opened the pull request:
:heavy_check_mark: Added User, Posts and Comments model
:heavy_check_mark: Added association between models
:heavy_check_mark: Added check new record validation

Torres-ssf commented on app/models/comment.rb:
```diff
@@ -0,0 +1,8 @@
+  belongs_to :user
+  belongs_to :post
+
+  validates :comment, presence: true
+  validates :user_id, presence: true
+  validates :post_id, presence: true
+end
```
- [ ] You are trying to validate a presence for a column that does not exist on your table, `:comment`. We need to change that for `:body` to make it possible to create a comment.

- [ ] There is no need to validate foreign keys. They are required by default.

Torres-ssf commented on app/models/post.rb:
```diff
@@ -0,0 +1,8 @@
+
+  validates :title, presence: true, uniqueness: true
+  validates :body, presence: true
+  validates :user_id, presence: true
```
- [ ] There is no need to validate foreign keys. They are required by default.

mcervantes71 commented:
:heavy_check_mark: Added node_modules to .gitignore
:heavy_check_mark: Changed validate a presence for a column that does exist on your table
:heavy_check_mark: Delete validation from foreign keys
